packages/millicast-sdk/src/Signaling.ts

```diff
@@ -229,29 +241,29 @@ export default class Signaling extends EventEmitter {
       if (optionsParsed.disableVideo && optionsParsed.disableAudio) {
         throw new Error('Not attempting to connect as video and audio are disabled')
       }
+      await this.connect()
```

Comment:
This was breaking tests all offering publishing stream tests and it took me a whole day to find lol. The TransactionManager is null until we try and connect. So we have to connect before the null check.